src/internal/restapi/v1/health/health.go.tmpl

```diff
@@ -31,6 +28,6 @@ func writeJson(ctx context.Context, w http.ResponseWriter, v interface{}) {
 	encoder.SetEscapeHTML(false)
 	err := encoder.Encode(v)
 	if err != nil {
-		logging.Ctx(ctx).Warn(fmt.Sprintf("error while encoding json response: %v", err))
```

Comment:
Significant loss of functionality, now it no longer logs with a TraceId. Let's do this when moving to a logging framework.

Reply:
How long do you wanna keep the current logging implementation before you want to replace it with the new one?

Reply:
- [ ] Restore logging component

Reply:
> How long do you wanna keep the current logging implementation before you want to replace it with the new one?

I usually prototype such changes in an actual service before adding them to the template. And it's better for the template to have some logging with trace ids than none. 

The moment we have a logging implementation that the team is happy with, all it takes is another PR in this template.